R/utils.R

```diff
@@ -376,36 +323,12 @@ conduct_parent_to_helpers <- function(api,
            ...)
     )
   } else {
-    out <- do.call(glue::glue("{function_identity}_sql"), arguments_to_pass)
+    out <- do.call(utils::getFromNamespace(function_identity, ns = "etnservice"),
```

Comment:
I would argue that some of the functions in `utils.R` are that big and crucial that they should be moved to their own file, e.g. `conduct_parent_to_helpers.R`. Doesn't necessarily need to be tackled in this PR.

Reply:
Sounds good to keep API helpers together. I would name the file `utils-api.R` or `utils-api_helpers.R`, cf. how `usethis::use_pipe()` creates a `utils-pipe.R`